execution/Cargo.toml

```diff
@@ -21,6 +21,8 @@ thiserror = "1.0.37"
 
 common = { path = "../common" }
 consensus = { path = "../consensus" }
+ic-http = { path = "../ic-http"}
+ic-cdk = "0.10.0"
```

Comment:
IMO this should be target guarded, unless if this is changed: https://github.com/eigerco/helios/blob/e42c2cda990f81c9e82aa300d96126b4f124f972/ic-http/src/http_request.rs#L14C1-L19

Reply:
removed ic-http, left ic-cdk unguarded